### ISSUE TITLE
Adaptive softmax transformer Language model replicate

### ISSUE THREAD

**Issue by liiutengbo** [pytorch/fairseq#1505]
Is there a standard configuration for language model for transformer_lm_wiki103,the official config can not replicate the result 18.7,but I can use the model to inference,and get the score.And my parameters num is 247.2M, but the offical model is 246.9M.

**Comment by myleott** [comment]
This doesn't follow the new issue templates, can you please open a new issue following one of the provided templates?
